packages/api/src/web3Middleware/provider.ts

```diff
@@ -2,5 +2,5 @@ import { Network } from './../types'
 
 export function getProvider (network: Network) {
   // The provider env variable name should be the network name specified in the config plus '_PROVIDER' ex.MOONBEAM_MOONBASE_PROVIDER
-  return process.env[`${network.toUpperCase().replace('-', '_')}_PROVIDER`]
+  return process.env[`${network.toUpperCase()}_PROVIDER`]
```

Comment:
With the current config file the provider return will be `CONFLUX-ESPACE-TESTNET_PROVIDER`. You need to update the config file or `${network.toUpperCase().split('-').join('_')}_PROVIDER` as the replace function only works with the first instance that matches '-'.

Reply:
Right, I thought we were using this function in a different context place. I'm going to close this issue and we'll remove the '-' in #197